api/pkg/auth/base.go

```diff
@@ -31,11 +33,22 @@ import (
 	auth "github.com/tektoncd/hub/api/pkg/auth/service"
 )
 
+// generate the random key if there is an error returns "SESSION_SECRET" as key
+func generateRandomKey(length int) string {
+	key := make([]byte, length)
+	_, err := rand.Read(key)
+	if err != nil {
+		return "SESSION_SECRET"
+	}
+	return base64.StdEncoding.EncodeToString(key)
```

Comment:
@pratap0007 Is there any restriction on the key length? encoding it to base64 may result unpredictable key length.

Reply:
maximum 64 bytes